### PR TITLE
VAGOV-TEAM-9189 VA Forms alert

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/9189

## QA steps
As an admin or content admin
1.Navigate to an existing va_form /find-forms/about-form-10-10143g
2. Click the edit tab.
image

  - [ ] Validate that the "Include Alert" field group is present  
  - [ ] Validate that option to "Place alert" is present.
  - [ ] Validate that the link to "create a new alert" indeed takes you to a place where an alert block can be created.
  - [ ] validate that the "Place alert" button allows you to choose an exiting alert.
3. Choose an alert block and save the node.   
  - [ ] Validate that the alert is visible on the node view.
4. Return to the  node edit tab.
  - [ ] validate that revision log message field is at the bottom of the page in the Editorial Workflow box
image

## Definition of Done
- [ ] QA steps pass
- [ ] Automated test pass.

### CODE REVIEW

File: config/sync/core.entity_form_display.node.landing_page.default.yml [hunk 1]
```diff
@@ -71,7 +71,6 @@ third_party_settings:
     group_editorial_workflow:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.landing_page.default.yml:74]: I need to fix this.  I don't know why it keeps moving back.

**Comment by swirtSJW** [comment]
I need to update behat tests for this change.

File: hooks/git/pre-commit [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 branch_name=$(git rev-parse --abbrev-ref HEAD)
-error_msg="Aborting commit. Your branch must start start with a VAGOV-* or VACMS-* Jira issue number format, e.g. 'VAGOV-123 or VAGOV-123-issue-name. Use \`git branch --move <VACMS-0000-new-name>\` to rename."
+error_msg="Aborting commit. Your branch must start with a VAGOV-TEAM-* or VACMS-* Github issue number format, e.g. 'VAGOV-TEAM-123-issue-name or VACMS-123-issue-name. Use \`git branch --move <VACMS-0000-new-name>\` to rename."
 # Regular expression to validate that the branch name starts with Jira story ID, e.g. VAGOV-000 or VACMS-000.
-no_match_vagov=1; [[ $branch_name =~ VAGOV-[0-9]+ ]] && no_match_vagov=0
+no_match_vagov=1; [[ $branch_name =~ VAGOV-TEAM-[0-9]+ ]] && no_match_vagov=0
 no_match_vacms=1; [[ $branch_name =~ VACMS-[0-9]+ ]] && no_match_vacms=0
 
 if [ $no_match_vagov == 1 ] && [ $no_match_vacms == 1 ]; then
```
Comment (ethanteague) [inline hooks/git/pre-commit:9]: @swirtSJW I like to add squashable commits for interactive rebasing, and it's a pain when these hooks run (and fail) on commits that aren't necessarily formatted with the intention of being used. Can we add something like ```suggestion
```#!/bin/sh
BRANCH_NAME=$(git branch | grep '*' | sed 's/* //')

if [ $BRANCH_NAME != '(no branch)' ]
then
  no_match_vagov=1; [[ $branch_name =~ VAGOV-TEAM-[0-9]+ ]] && no_match_vagov=0
  no_match_vacms=1; [[ $branch_name =~ VACMS-[0-9]+ ]] && no_match_vacms=0
fi`
```

Comment (swirtSJW) [reply hooks/git/pre-commit:9]: Maybe I am misunderstandng..  You want the option of not working on a named branch?

Comment (ethanteague) [reply hooks/git/pre-commit:9]: I want the check to not run when I am in the middle of a rebase, b/c I might have commits that are something like: `squash me` - this explains better, @swirtSJW https://gmurphey.github.io/2013/02/02/ignoring-git-hooks-when-rebasing.html

Comment (ElijahLynn) [reply hooks/git/pre-commit:9]: Did your suggestion get borked? I don't see the suggestion.

Comment (ethanteague) [reply hooks/git/pre-commit:9]: Yeah, it was whacked - just fixed, @ElijahLynn :+1:

Comment (swirtSJW) [reply hooks/git/pre-commit:9]: That test runs on the branch name not on the commit message.
It should never alarm in a  rebase.  I use interactive rebase all the time and have never had an issue with this.

Comment (ethanteague) [reply hooks/git/pre-commit:9]: :man_shrugging: - alrighty. It catches me whenever I try to run the `F` fixup to squash down a message that doesn't follow the `VACMS-000: Some message` format. No big deal, @swirtSJW

**Pull request merged**